internal/distro/fedora/distro.go

```diff
@@ -160,14 +160,15 @@ var (
 		defaultImageConfig: &distro.ImageConfig{
 			Locale: common.ToPtr("en_US.UTF-8"),
 		},
-		defaultSize:         10 * common.GibiByte,
-		rpmOstree:           true,
-		bootable:            true,
-		image:               iotRawImage,
-		buildPipelines:      []string{"build"},
-		payloadPipelines:    []string{"image-tree", "image", "xz"},
-		exports:             []string{"xz"},
-		basePartitionTables: iotBasePartitionTables,
+		defaultSize:            4 * common.GibiByte,
+		rpmOstree:              true,
+		bootable:               true,
+		image:                  iotRawImage,
+		buildPipelines:         []string{"build"},
+		payloadPipelines:       []string{"image-tree", "image", "xz"},
+		exports:                []string{"xz"},
+		basePartitionTables:    iotBasePartitionTables,
+		requiredPartitionSizes: map[string]uint64{},
```

Comment:
What about a small comment above this line to highlight that the empty map is actually needed here? It might make the code less surprising for future selfs?

Reply:
Done.